shell/demo/demo2.ps1

```diff
@@ -0,0 +1,173 @@
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory=$true, HelpMessage="Prelude Token")]
+    [String]$preludeToken
+)
+
+function FromEnv { param ([string]$envVar, [string]$default)
+    $envVal = [Environment]::GetEnvironmentVariable($envVar, "User")
+    if ($envVal) { return $envVal } else { return $default }
+}
+
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+$PRELUDE_API = FromEnv "PRELUDE_API" "https://api.preludesecurity.com"
+
+$TEST_ID="2e705bac-a889-4283-9a8e-a12358fa1d09"
+$TEST_INTRO="Will your computer quarantine a malicious ELF file?
+
+Malicious files are used to gain entry and conduct cyberattacks against corporate systems through seemingly
+innocuous email attachments or direct downloads. For example - a malicious Linux executable was used by the Royal Ransomware group 
+during a ransomware attack in February 2023.
+
+This test will attempt to download a malicious file to your disk - a defanged version of the Royal Ransomware ELF binary - 
+to see how your machine will respond. Downloading malicious files can open you up to further attacks,
```

Comment:
Let's leave this. In the future, I wonder if we should use verbiage such as `how your defenses will respond` in the future.